code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -89,6 +92,15 @@
 	summoner << "<span class='danger'><B>Your [name] died somehow!</span></B>"
 	summoner.death()
 
+/mob/living/simple_animal/hostile/guardian/proc/updatehudhealth()
+	if(summoner)
+		var/resulthealth
+		if(iscarbon(summoner))
+			resulthealth = round((summoner.health / abs(config.health_threshold_dead - summoner.maxHealth)) * 100)
+		else
+			resulthealth = round((summoner.health / summoner.maxHealth) * 100)
+		hud_used.healthdisplay.maptext = "<div align='center' valign='middle' style='position:relative; top:0px; left:6px'><font color='#efeeef'>[resulthealth]%</font></div>"
```

Comment:
I have no fucking idea, this is copied from blob health/power displays and I don't think there's another way to center it but I know nothing about this